rosdep/python.yaml

```
@@ -3373,6 +3373,16 @@ python-pyzbar-pip:
   ubuntu:
     pip:
       packages: [pyzbar]
+python-qpsolvers-pip:
```

Our current best practice is to make these keys prefixed with `python3`:

```suggestion
python3-qpsolvers-pip:
```

(note that you'll also have to move this block so that it is in alphabetical order)